### PR TITLE
Feat/return pkg info on image

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Added packages information to (updated, removed, added, total installed, aditional) to image endpoint.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Documentation update
- [X] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [x] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/routes/images.go [hunk 1]
```diff
@@ -353,10 +354,39 @@ func GetImageStatusByID(w http.ResponseWriter, r *http.Request) {
...
 
+//ImageDetail return the structure to inform package info to images
+type ImageDetail struct {
+	Image             *models.Image `json:"Image"`
+	AditionalPackages int           `json:"AditionalPackages"`
+	Packages          int           `json:"Packages"`
+	UpdateAdded       int           `json:"UpdateAdded"`
+	UpdateRemoved     int           `json:"UpdateRemoved"`
+	UpdateUpdated     int           `json:"UpdateUpdated"`
+}
```
Comment (Avielyo10) [inline pkg/routes/images.go:365]: Please use snake_case instead of CamelCase on JSON tags

File: pkg/routes/images_test.go [hunk 2]
```diff
@@ -137,14 +137,35 @@ func TestGetStatus(t *testing.T) {
 	}
 }
 
+type ImageResponse struct {
+	Image             models.Image `json:"Image"`
+	AditionalPackages int          `json:"AditionalPackages"`
+	Packages          int          `json:"Packages"`
+	UpdateAdded       int          `json:"UpdateAdded"`
+	UpdateRemoved     int          `json:"UpdateRemoved"`
+	UpdateUpdated     int          `json:"UpdateUpdated"`
```
Comment (Avielyo10) [inline pkg/routes/images_test.go:146]: Please use snake_case instead of CamelCase on JSON tags

File: pkg/routes/imagesets.go [hunk 3]
```diff
@@ -164,10 +159,18 @@ func ListAllImageSets(w http.ResponseWriter, r *http.Request) {
 
 }
 
+//ImageSetImagePackages return info related to details on images from imageset
+type ImageSetImagePackages struct {
+	ImageSetData     models.ImageSet `json:"ImageSet"`
+	Images           []ImageDetail   `json:"Images"`
+	ImageBuildISOURL string          `json:"ImageBuildISOURL"`
```
Comment (Avielyo10) [inline pkg/routes/imagesets.go:166]: Please use snake_case instead of CamelCase on JSON tags

File: pkg/services/images.go [hunk 4]
```diff
@@ -652,6 +654,40 @@ func (s *ImageService) calculateChecksum(isoPath string, image *models.Image) er
...
 }
 
+//ImageDetail return the structure to inform package info to images
+type ImageDetail struct {
+	Image             *models.Image `json:"Image"`
+	AditionalPackages int           `json:"AditionalPackages"`
+	Packages          int           `json:"Packages"`
+	UpdateAdded       int           `json:"UpdateAdded"`
+	UpdateRemoved     int           `json:"UpdateRemoved"`
+	UpdateUpdated     int           `json:"UpdateUpdated"`
```
Comment (Avielyo10) [inline pkg/services/images.go:664]: Please use snake_case instead of CamelCase on JSON tags

**Review by Avielyo10 — CHANGES_REQUESTED** [review]
For new structs please use snake_case instead of CamelCase on JSON tags

File: pkg/routes/images_test.go [hunk 5]
```diff
@@ -154,20 +175,41 @@ func TestGetImageById(t *testing.T) {
...
 	}
 
-	var ir models.Image
+	var ir ImageResponse
 	respBody, err := ioutil.ReadAll(rr.Body)
 	if err != nil {
 		t.Errorf(err.Error())
 	}
 
+	fmt.Printf(":: respBody :: %v\n", &ir)
```
Comment (biancarosa) [inline pkg/routes/images_test.go:184]: 👀

File: pkg/routes/imagesets.go [hunk 6]
```diff
@@ -242,16 +256,24 @@ func contains(s []string, searchterm string) bool {
...
-		img, err := s.ImageService.GetImageByID(id)
 		if err != nil {
 			log.Error("Image detail not found \n")
 		}
-		Imgs = append(Imgs, *img)
+		Imgs = append(Imgs, ImageDetail(img))
+		fmt.Printf("::: list last element:: %v\n", Imgs[len(Imgs)-1].Image.ID)
 	}
+
+	fmt.Printf("::: list :: %v\n", Imgs)
```
Comment (biancarosa) [inline pkg/routes/imagesets.go:277]: 👁️

File: pkg/routes/imagesets.go [hunk 7]
```diff
@@ -242,16 +256,24 @@ func contains(s []string, searchterm string) bool {
...
 
-	for _, image := range imageSet.Images {
-		id := strconv.FormatUint(uint64(image.ID), 10)
-		img, err := s.ImageService.GetImageByID(id)
 		if err != nil {
 			log.Error("Image detail not found \n")
 		}
-		Imgs = append(Imgs, *img)
+		Imgs = append(Imgs, ImageDetail(img))
+		fmt.Printf("::: list last element:: %v\n", Imgs[len(Imgs)-1].Image.ID)
```
Comment (biancarosa) [inline pkg/routes/imagesets.go:274]: 👀

**Review by biancarosa — APPROVED** [review]
I'm good with it.

**Comment by acosferreira** [comment]
@Avielyo10 I'm merging to unblock FE Team, let me know if you see anything else that need to be changed or fixed.

**Pull request merged**